docs/software/spack.md

```diff
@@ -0,0 +1,179 @@
+
+```
+[johndoe@c3cpu-c11-u17-2 ~]$ spack list fastqc
+fastqc
+==> 1 packages
+```
+
+>For additional information about a specific piece of software (available versions, variants, and dependencies), you can use the `spack info <software name>` command. Note that the 'preferred version' is the default version which will be installed if no alternative version is specified. 
+
+Once you've confirmed that a piece of software is available, you can install it in the active environment using the `spack install --add` command:
+
+```
+[johndoe@c3cpu-c11-u17-2 ~]$ spack install --add fastqc
+```
+>Note that the `--add` argument is required to add package specs to an environment. You can separately add specs without installing a package using the `spack add <software name>` command.
+
+This will install the specified software and all dependencies into your environment.
+
+You can also modify the installation commands to meet your installation needs. You can specify which version you'd like installed using the `@` operator:
+
+```
+[johndoe@c3cpu-c11-u17-2 ~]$ spack install --add fastqc@0.11.9
+```
+
+#### Installation Locations and Specifics
+
+The root of the Spack installation tree, by default, is `/projects/$USER/software/spack` (this can be changed in `~/config.yaml`). From there, executables can be found in subdirectories depending on the OS and compiler used to install the software. Software will be installed here regardless of whether the software was installed in an environment. 
+
+Spack will always check the installation root prior to installing software, in an environment or otherwise. If the software is not found in the installation root, it will be installed there. If the software is found in the installation root, it will not be installed. When attempting to install in an environment, symbolic links to the installation root will be created in `/projects/$USER/spack/environments/<envname>/.spack-env/view`. This behavior means that once a piece of software is installed, it can be linked to any environment without having to perform the installation again. 
+
+The default cache is located at `/scratch/alpine/$USER/spack/cache`. This directory will contain any previously-installed tarballs and repositories, as well as any miscellaneous cache items.  
+
+### Installing and Using Compilers with Spack
+
+In addition to standard software packages, you can use Spack to install compilers which are not currently available in the Alpine software stack. For example, if you need the newest available version of `gcc`, you can install it into your environment as follows: 
+
+```
+[johndoe@c3cpu-c11-u17-2 ~]$ spack install --add gcc@13.1.0
+```
+
+Once the installation is complete, you can make it available to Spack with `spack compiler add`:
+
+```
+[johndoe@c3cpu-c11-u17-2 ~]$ spack compiler add <path to compiler>
+```
+
+>Note that, in this case, you can locate the path of the compiler with `spack location -i gcc@13.1.0`.
```

Comment:
```suggestion
```

I believe you can remove this portion because I use and explain these commands in my suggested steps above.